client/marketing/overview/installed-extensions/index.js

```diff
@@ -6,14 +6,14 @@ import { __ } from '@wordpress/i18n';
 import { compose } from '@wordpress/compose';
 import { withSelect, withDispatch } from '@wordpress/data';
 import PropTypes from 'prop-types';
-import { Card } from '@woocommerce/components';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 import InstalledExtensionRow from './row';
 import { STORE_KEY } from '../../data/constants';
+import { Card } from '../../components';
 
 class InstalledExtensions extends Component {
```

Comment:
It looks like the border colour for each row in this card could be updated to match the border color at the bottom of the card header.

![](https://d.pr/i/WaXuPI+)